internal/vppinit/vppinit.go

```diff
@@ -157,6 +166,53 @@ func createAfPacket(ctx context.Context, vppConn api.Connection, link netlink.Li
 	return afPacketCreateRsp.SwIfIndex, nil
 }
 
+func addArp(ctx context.Context, vppConn api.Connection, swIfIndex interface_types.InterfaceIndex) error {
+	f, err := os.OpenFile("/proc/net/arp", os.O_RDONLY, 0600)
```

Comment:
Please use:

https://pkg.go.dev/github.com/vishvananda/netlink#NeighList

To get the list of both IPv4 and IPv6 Neighbors.

Reply:
Also... we should probably take some care to only add ARP entries that are in state [PERMANENT](https://github.com/vishvananda/netlink/blob/v1.1.0/neigh_linux.go#L40).  I presume these AWS neighbor entries are in state PERMANENT?

Reply:
You are right! Fixed